### PR TITLE
repair back ticks

### PR DESCRIPTION
**Pull request by kghamilton89**

### CODE REVIEW

File: SocrataIowaCompensation/README.md [hunk 1]
```diff
@@ -277,7 +277,8 @@ Navigate to **Unemployment insurance claims and payments (statewide - monthly)**
 
 As with the previous figure, we can see that this dataset follows the same general trend, with the payments tending to peak in January of a given year and continue to fall throughout
 the rest of the year. However, we can see that in the late 2000's, the series shifted up as marked in the above figure. We can also add a 'shifted' duplicate of this series.
-```
+
+```sh
```
Comment (rodionos) [inline SocrataIowaCompensation/README.md]: For chartlab syntax, use `ls`.

File: SocrataIowaCompensation/README.md [hunk 2]
```diff
@@ -366,7 +367,7 @@ The setup for this example is almost exactly the same as the previous example, e
 from the previous example. To calculate our the compensation per employee value, we considered 'local' and 'state' employees to be one and the same. Consequently, we simply divided the total compensation
 by the average of 'emps' (alias for state government employees) and 'empl' (alias for local government employees). This line in the configuration is shown below:
 
-```
+```txt
```
Comment (rodionos) [inline SocrataIowaCompensation/README.md]: For functions, use `javascript`

File: Support/Moving-Avg/README.md [hunk 3]
```diff
@@ -104,7 +104,7 @@ produces the following:
 
 **Scale Modification**:
 
-```enplanements = value/100```
```
Comment (rodionos) [inline Support/Moving-Avg/README.md]: For functions and formulas, use `javascript`

File: Support/Moving-Avg/README.md [hunk 4]
```diff
@@ -114,7 +114,7 @@ produces the following:
 
 **Scale Modification**:
 
-```enplanements = value/10000```
+`enplanements = value/10000`
```
Comment (rodionos) [inline Support/Moving-Avg/README.md]: For functions and formulas, use `javascript`

File: IllinoisBirthrates/README.md [hunk 5]
```diff
@@ -476,9 +476,7 @@ Using [Fityk](http://fityk.nieto.pl/) to create a best-fit model for this data:
 
 The associated formula is shown below:
 
-``
-F(x) = 197 + -2.59*x + 0.179*x^2 + -0.00511*x^3
-``
+`F(x) = 197 + -2.59*x + 0.179*x^2 + -0.00511*x^3`
```
Comment (rodionos) [inline IllinoisBirthrates/README.md]: For functions and formulas, use `javascript` block

File: IllinoisBirthrates/README.md [hunk 6]
```diff
@@ -492,7 +490,7 @@ Excluding the final data point from the series, which deviated significantly, cr
 
 This model's formula is:
 
-``F(x) = 196 + -1.7*x + 0.0587*x^2 + -0.000794*x^3``
+`F(x) = 196 + -1.7*x + 0.0587*x^2 + -0.000794*x^3`
```
Comment (rodionos) [inline IllinoisBirthrates/README.md]: For functions and formulas, use `javascript`

File: IllinoisBirthrates/README.md [hunk 7]
```diff
@@ -527,7 +525,7 @@ When updated to include the latest figures, the model looks like this:
 
 This updated model's formula is shown here:
 
-``F(x) = 197 + -2.53*x + 0.189*x^2 + -0.006*x^3``
+`F(x) = 197 + -2.53*x + 0.189*x^2 + -0.006*x^3`
```
Comment (rodionos) [inline IllinoisBirthrates/README.md]: For functions and formulas, use `javascript` block

**Comment by kghamilton89** [comment]
Requested changes implemented.

**Pull request merged**